generator/postgres/postgres_generator.go

```diff
@@ -69,7 +69,7 @@ func GenerateDSN(dsn, schema, destDir string, templates ...template.Template) (e
 }
 
 func openConnection(dsn string) *sql.DB {
-	fmt.Println("Connecting to postgres database: " + dsn)
+	fmt.Printf("Connecting to postgres database: %v...", dsn[:15])
```

Comment:
I think we can just remove `+ dsn` part.

Reply:
The same for `mysql`.

Reply:
Yeah that would be the best